slink-main/config.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const awsParamStore = require('aws-param-store');
+
+const configParams = {
+  SAP_ADD_EMPLOYEE_URL: {
+    name: 'sap/ADD_EMPLOYEE_URL',
+    value: undefined
+  },
+  SAP_USERNAME: {
+    name: 'sap/USERNAME',
+    value: undefined
+  },
+  SAP_PASSWORD: {
+    name: 'sap/PASSWORD',
+    value: undefined
+  },
+  SR_ADD_PROP_URL: {
+    name: 'smartrecruiters/ADD_PROP_URL',
+    value: undefined
+  },
+  SR_EMPLOYEE_PROP_ID: {
+    name: 'smartrecruiters/EMPLOYEE_PROP_ID',
+    value: undefined
+  },
+  SR_JOB_PROPS_URL: {
+    name: 'smartrecruiters/JOB_PROPS_URL',
+    value: undefined
+  },
+  SR_SUMMARY_URL: {
+    name: 'smartrecruiters/SUMMARY_URL',
+    value: undefined
+  },
+  SR_TOKEN: {
+    name: 'smartrecruiters/TOKEN',
+    value: undefined
+  }
+};
+
+/**
+ * Helper function to call AWS API to retrieve parameter values
+ * Extracting this call in its own function also helps in mocking tests
+ * @param paramPath
+ * @returns Parameter keys/values raw JSON
+ */
+const getAWSParams = async (paramPath) => {
+  // Get the AWS region from the environment variable which gets created automatically by AWS
+  let awsRegion = process.env.AWS_DEFAULT_REGION;
+
+  // the environment doesn't know which AWS region we are running in so default it to us-east-1
+  if (awsRegion === undefined) {
+    awsRegion = 'us-east-1';
+  }
+
+  const parameters = await awsParamStore.getParametersByPath(paramPath, { region: awsRegion });
+  return parameters;
+};
+
+// This is an awful work around to mocking a single function using jest!!
+// Apparently mocking a single function in a module doesn't work!
+const lib = {
+  getAWSParams,
+};
```

Comment:
See `introduction_test.js:12`.  Worked for me, but possibly because the code/test didn't care about the rest of util.js (?).